### PR TITLE
Fixes #261 - Added note for oversized emails / small changes

### LINKED ISSUE #261: Monitoring documentation outdated
The documentation still mentions a token returned by the monitoring endpoint - that was removed with Zammad 6.0.
It may also make sense to mention that, while "unprocessible mail" are being noted, "oversized mail" are not.

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

**Comment by ralf401** [comment]
Hi @mgruner, can you have a brief look? I just added a note for oversized emails (and converted one existing note) , adjusted (cropped) one image and inserted some linebreaks.

Thank you!

File: system/monitoring.rst [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 **********
 
-**Please note:** This is only available in self hosted instances, as we're monitoring hosted instances and fix problems.
+.. note:: This is only available in self hosted instances, as we're
+   monitoring hosted instances and fix problems.
 
 On the monitoring page you can see the current health state of Zammad.
-This can be useful if you for example have the feeling that you don't receive emails anymore, you can take a look here before logging onto your Server.
+This can be useful if you for example have the feeling that you don't receive
+emails anymore, you can take a look here before logging onto your Server.
```
Comment (mgruner) [inline system/monitoring.rst:9]: Server -> server

File: system/monitoring.rst [hunk 2]
```diff
@@ -20,3 +23,9 @@ Zammad has issue (whatever nature, refer to image 2 for interface example)::
 
 .. image:: /images/system/monitoring/1.jpg
```
Comment (mgruner) [inline system/monitoring.rst:24]: I suggest to reorder this part as well. First should come the text and image where things are green, and then the part where there is an alert (or the other way round). Not first all text with "refer to image 2".

**Review by mgruner — COMMENTED** [review]
Sorry Ralf, I comment on the text that you didn't change. Do you think we have a chance of improving it? Let me know if anything is unclear.

**Review by mgruner — APPROVED** [review]
Looks nice!

**Pull request merged**